neuro_evolution_ctrnn/optimizer/optimizer_mu_lambda.py

```diff
@@ -20,21 +19,25 @@ class OptimizerMuPlusLambda(IOptimizer[OptimizerMuLambdaCfg]):
 
     def __init__(self, eval_fitness: Callable, individual_size: int, random_seed: int, conf: OptimizerMuLambdaCfg,
                  stats, map_func=map, from_checkpoint=None, reset_hof=False):
-        super(OptimizerMuPlusLambda, self).__init__(eval_fitness, individual_size, random_seed, conf, stats, map_func,
-                                                    from_checkpoint)
-        toolbox = self.toolbox
+        super(OptimizerMuPlusLambda, self).__init__(eval_fitness, random_seed, conf, stats, map_func,
+                                                    from_checkpoint, reset_hof)
 
         if self.conf.strategy_parameter_per_gene:
             individual_size *= 2
         else:
             # add two genes for strategy parameters used in mutate
             individual_size += 2
```

Comment:
Siehe mein kommentar bei `strip_strategy_from_population`

```suggestion
        if self.conf.strategy_parameter_per_gene:
            # add a gene as strategy parameters for each individual actual gene
            self.number_strategy_params_in_genome = individual_size 
        else:
            # add two genes as strategy parameters for the entire genome
            self.number_strategy_params_in_genome += 2 
        individual_size += self.number_strategy_params_in_genome
```